apps/site/assets/ts/schedule/components/line-diagram/StopCard.tsx

```diff
@@ -78,7 +78,12 @@ const StopCard = (props: StopCardProps): ReactElement<HTMLElement> => {
   const showPrediction = hasLivePredictions && !isDestination;
   const showDiversion =
     diversionAlert && !(hasLivePredictions && isDestination);
-
+  const showDepartures = (): string => {
+    if (routeStop.route?.type === 1 || routeStop.route?.type === 0) {
+      return liveData?.headsigns.length ? "View upcoming departures" : "";
```

Comment:
suggestion: Using an empty string for a button label feels like something you'd never want. Maybe is this conditional not even necessary? I.e., can we just return "View upcoming departures" for all subway routes?